### PR TITLE
Add payment info to `offline_transaction_charge.created`

### PR DESCRIPTION
**Pull request by awadyas**
Closes issue https://github.com/CommitChange/tix/issues/3944

### CODE REVIEW

File: app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder [hunk 1]
```diff
@@ -2,9 +2,15 @@
 
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+json.id paymentable.houid
```
Comment (wwahammy) [inline app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder]: I don't understand why this is needed. It appears to be done in [app/views/api_new/subtransaction_payments/_subtransaction_payment.json.jbuilder](https://github.com/CommitChange/houdini/blob/fa4ed7538df2167f44cb0ae48490c426102245c5/app/views/api_new/subtransaction_payments/_subtransaction_payment.json.jbuilder#L7). Is that not working?

File: app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder [hunk 2]
```diff
@@ -2,9 +2,15 @@
...
 
 json.object 'offline_transaction_charge'
 
+json.created paymentable.created.to_i
```
Comment (wwahammy) [inline app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: Same question as above.

**Review by wwahammy — CHANGES_REQUESTED** [review]
Thanks @awadyas! I had a question here about why some code is in app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder

File: app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder (hunk 1 above)
Comment (awadyas) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder]: Ah, I see what youre saying

File: app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder (hunk 2 above)
Comment (awadyas) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: This one is also in the subtransaction_payment jbuilder, but it's recording the datetime for the subtransaction_payment, not the paymentable. Won't that be a problem?
```ruby
json.created subtransaction_payment.created.to_i
```

Comment (awadyas) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: / can we assume the time or creation will be the same?

Comment (awadyas) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: @wwahammy what are your thoughts on this?

Comment (wwahammy) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: Let's use `json.created paymentable.created.to_i` and we should document the weirdness. Please file an issue to explain the weird behavior here.

Comment (awadyas) [reply app/views/api_new/offline_transaction_charges/_offline_transaction_charge.json.jbuilder:8]: Done

**Review by wwahammy — APPROVED** [review]

**Pull request merged**